include/matx_scalar_ops.h

```diff
@@ -133,11 +101,32 @@ template <typename T1, typename T2, typename T3, typename F> class TerOp {
   using scalar_type = std::invoke_result_t<decltype(op), T1, T2, T3>;
 };
 
-MATX_UNARY_OP_GEN(ceil, Ceil);
-MATX_UNARY_OP_GEN(floor, Floor);
-MATX_UNARY_OP_GEN(round, Round);
-MATX_UNARY_OP_GEN(sqrt, Sqrt);
-MATX_UNARY_OP_GEN(exp, Exp);
+
+MATX_UNARY_OP_GEN(ceil, Ceil, zero);
+MATX_UNARY_OP_GEN(floor, Floor, zero);
```

Comment:
zero?

Reply:
Ceiling and floor have zero for a derivative no?

Reply:
That makes sense. I guess it's something that wouldn't be used.